src/main/webapp/community.html

```diff
@@ -12,12 +12,12 @@
         fetch(url).then((response) => {
           return response.json();
         }).then((users) => {
-          const list = document.getElementById('list');
```

Comment:
are the two 'list' in the line the same? I am not sure but I changed both of them to 'userlist'